src/data/mv_polynomial.lean

```diff
@@ -627,6 +637,14 @@ finset.sup_le $ assume b,
 
 end rename
 
+lemma eval₂_cast_comp {β : Type u} {γ : Type v} [decidable_eq β] [decidable_eq γ] (f : γ → β)
+  {α : Type w} [comm_ring α] (g : β → α) (x : mv_polynomial γ ℤ) :
+  eval₂ int.cast (g ∘ f) x = eval₂ int.cast g (rename f x) :=
+mv_polynomial.induction_on x
```

Comment:
It should be possible to prove this without a separate induction, using `eval₂_hom_X`, but I couldn't figure out how to deal with `[is_ring_hom]`.

Reply:
The current proof is pretty straightforward: I'm not too sad that it isn't maximally refactored. Unless you think it's important, I'll mark this as resolved for now.